workspace.jsonc

```diff
@@ -612,6 +612,12 @@
     "scopes/code": {
       "defaultScope": "teambit.code"
     },
+    "scopes/readme": {
```

Comment:
once you moved the component (see comment on the bitmap file)
the rule here should be `scopes/mdx/readme`
also, you won't need the `default scope` here anymore, as everything under the `scopes/mdx` is going to mdx scope.